### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by shivampandey0**
Screenshot 2022-01-29 at 7 24 25 PM

Completed alert components
- Simple alert
- Alert with a link
- Alert with icon
- Alert with close button

### CODE REVIEW

**Review by Hamdrive — COMMENTED** [review]
LGTM

File: css/components/alerts.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.alert {
+  position: relative;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border: 2px solid transparent;
```
Comment (Hamdrive) [inline css/components/alerts.css]: update border to show styling, else if redundant then remove

File: css/components/alerts.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  color: var(--text-black);
+}
+
+.alert-link {
+  color: inherit;
+  font-weight: bold;
+}
+
+.alert-icon {
+  margin-right: 0.5rem;
```
Comment (Hamdrive) [inline css/components/alerts.css]: Use px instead of rem for such small values

File: css/components/alerts.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  height: 1.5rem;
+  border: none;
+  position: absolute;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  outline: none;
+  opacity: 0.5;
+  cursor: pointer;
+  margin: 0.75rem;
```
Comment (Hamdrive) [inline css/components/alerts.css:57]: Keep it in px, ideally if its above 10px then use rem

File: css/base/base.css [hunk 4]
```diff
@@ -13,6 +13,12 @@
 
   --grey: hsl(207, 21%, 48%);
   --dark-grey: hsl(207, 25%, 28%);
+
+  --alert-primary: 	hsl(239, 84%, 67%);
+  --alert-secondary: hsl(0, 8%, 60%);
+  --alert-success: hsl(153, 53%, 53%);
```
Comment (ritik-patel05) [inline css/base/base.css]: can rename to success because generally over the whole website, it will have the same color.

File: css/base/base.css [hunk 5]
```diff
@@ -13,6 +13,12 @@
...
+  --alert-danger: hsl(348, 86%, 61%);
```
Comment (ritik-patel05) [inline css/base/base.css]: rename: danger

File: css/base/base.css [hunk 6]
```diff
@@ -13,6 +13,12 @@
...
+  --alert-warning: hsl(44, 100%, 77%);
```
Comment (ritik-patel05) [inline css/base/base.css]: rename: warning

File: css/components/alerts.css (hunk 1 above)
Comment (shivampandey0) [reply css/components/alerts.css]: removed unnecessary styling

File: css/components/alerts.css (hunk 2 above)
Comment (shivampandey0) [reply css/components/alerts.css]: removed rem for smaller units

**Review by ritik-patel05 — COMMENTED** [review]
LGTM

File: css/components/alerts.css (hunk 3 above)
Comment (shivampandey0) [reply css/components/alerts.css:57]: done

File: css/base/base.css (hunk 4 above)
Comment (shivampandey0) [reply css/base/base.css]: done

File: css/base/base.css (hunk 5 above)
Comment (shivampandey0) [reply css/base/base.css]: done

File: css/base/base.css (hunk 6 above)
Comment (shivampandey0) [reply css/base/base.css]: done

**Pull request merged**